src/pybind/mgr/volumes/module.py

```diff
@@ -148,6 +148,16 @@ class Module(orchestrator.OrchestratorClientMixin, MgrModule):
             'desc': "List auth IDs that have access to a subvolume",
             'perm': 'r'
         },
+        {
+            'cmd': 'fs subvolume evict '
+                   'name=vol_name,type=CephString '
+                   'name=sub_name,type=CephString '
+                   'name=auth_id,type=CephString '
+                   'name=group_name,type=CephString,req=false '
+                   'name=use_subvol_path,type=CephBool,req=false ',
+            'desc': "Evict clients based on auth IDs and subvolume mounted",
```

Comment:
isn't it always better to evict a client based on subvol_path and auth ID? I see that the ceph_volume_client's evict() had an argument volume_path=None. It might be confusing for the end use to understand the nuance of 'use_subvol_path'. Since authorize CLI authorizes an auth ID to access to a particular subvolume, I think evict CLI should perform similar action where it evicts a client that has mounted the subvolume with the auth ID.

Reply:
I initially did that way and later checked ceph_volume_client and replicated the behavior. So shall I remove evicting client without subvolume path option and retain single behaviour of always using subvolpath ?

Reply:
Done